Build.fs

```diff
@@ -5,7 +5,7 @@ open Farmer.Builders
 
 open Helpers
```

Comment:
Helpers has had some changes over time, most notably using CreateProcess.FromRawCommand instead of CreateProcess.FormRawCommandLine. This also reflects in this file, where commands are invoked with strings instead of a list of arguments. It's mostly a stylistic thing, I think it would be nice to keep the dojo in sync.